src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -123,6 +126,9 @@ def neighbor_set (G' : subgraph G) (v : V) : set V := set_of (G'.adj v)
 lemma neighbor_set_subset (G' : subgraph G) (v : V) : G'.neighbor_set v ⊆ G.neighbor_set v :=
 λ w h, G'.adj_sub h
 
+lemma neighbor_set_subset_verts (G' : subgraph G) (v : V) : G'.neighbor_set v ⊆ G'.verts :=
+by { intros _ h, exact G'.edge_vert (adj_symm G' h) }
```

Comment:
Once you get down to `intros ..., exact ...`, you may as well turn it into a proof term.
```suggestion
λ _ h, G'.edge_vert (adj_symm G' h)
```
You don't strictly need to, though.